### PR TITLE
[MIG] multi step wizard: Migration to 16.0

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

**Review by AntoniRomera — APPROVED** [review]

**Review by MRomeera — APPROVED** [review]
Ok

**Comment by FernandoRomera** [comment]
@OCA/tools-mainteiners
Can you merge this PR, please?

File: multi_step_wizard/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import test_multi_step_wizard
+from . import common
```
Comment (simahawk) [inline multi_step_wizard/tests/__init__.py:4]: FTR the import of common here is useless

**Review by simahawk — APPROVED** [review]

**Pull request merged**